cfme/infrastructure/host.py

```diff
@@ -164,13 +164,8 @@ def delete(self, cancel=True):
         Args:
             cancel (bool): Whether to cancel the deletion, defaults to True
         """
-
         view = navigate_to(self, "Details")
-        view.toolbar.configuration.item_select("Remove item", handle_alert=cancel)
-        if not cancel:
-            view = self.create_view(HostsView)
-            assert view.is_displayed
-            view.flash.assert_success_message("The selected Hosts / Nodes was deleted")
+        view.toolbar.configuration.item_select("Remove item", handle_alert=not cancel)
```

Comment:
I've returned it back. The reason I had deleted it before is that it hadn't work for Openstack nodes. Fixed it [here](https://github.com/ManageIQ/integration_tests/pull/5152/files#diff-48da398b86a224ed0b471e7540050346R42)